src/datasources/fault-ds/partials/config.html

```diff
@@ -1,2 +1,44 @@
 <datasource-http-settings current="ctrl.current">
 </datasource-http-settings>
+
+<h3 class="page-heading">OpenNMS FM Datasource Details</h3>
+<div class="gf-form-group">
+    <div class="gf-form-inline">
+        <gf-form-switch class="gf-form"
+                        label="Use grafana user"
+                        tooltip="Defines if on supported alarm operations the currently logged in
```

Comment:
I would rephrase this as:
Used to control whether operations on alarms are performed the data source user, or the user that is currently logged in to Grafana.